src/components/RCLink/RCLink.tsx

```diff
@@ -30,22 +30,23 @@ const RCLink = ({
   ...rest
 }: RCLinkProps) => {
   return (
-    <Link href={href} passHref>
-      <DSLink
-        className={className}
-        fontWeight={active && "bold"}
-        {...rest}
-        __css={
-          hasWhiteFocusRing && {
-            _focus: {
-              outlineColor: "ui.white",
-            },
-          }
+    // <Link href={href} passHref>
```

Comment:
This is temporary. The Nextjs and DS Link components aren't working together.

Reply:
ruh roh. is that something to do with the middleware?

Reply:
With the DS. I'll update this independently.